### ISSUE TITLE
Is this just guessing?

### ISSUE THREAD

**Issue by julioasotodv** [XLabs-AI/x-flux#33]
Given that official training code for Flux has not been released, how can you ensure that this training code is even remotely close to the official unreleased one? I recall someone at BFL saying that training loss and procedure was unlike anything released yet…

Is this repo just guessing, or is there any resource detailing how the model should be trained?

Thank you!

**Comment by gradient-diffusion** [comment]
The flux dev is just flow matching and we are using loss from flow mathching

**Comment by julioasotodv** [comment]
> The flux dev is just flow matching and we are using loss from flow mathching

Apparently it is flow matching + a ton of small, non-described tricks throughout the way during training; so it may be the reason why fine-tunes are finicky
